### ISSUE TITLE
3.6.1 When decompiling the script, the label fails to display normally

### ISSUE THREAD

**Issue by XMDS** [sannybuilder/dev#99]
@x87 I noticed at compile time. The label is compiled into a numeric offset
A simple example：

```
{$CLEO .cs}
0000: NOP
jump @1

:1
xxxx() 
0001: wait 0 ms 
00D6: if and
0256:   player $PLAYER_CHAR defined 
03EE:   player $PLAYER_CHAR controllable 
004D: jump_if_false @1 
0226: 0@ = actor $PLAYER_ACTOR health 
0AD1: show_formatted_text_highpriority "HEALTH: %d" time 3000 0@ 
0A93: terminate_this_custom_script 

:xxxx
015D: set_gamespeed 1.0
return
```
Decompile:
```
{$CLEO .cs}

0000: NOP 
0002: jump -9 
0050: gosub -72 
0001: wait 0 ms 
00D6: if and
0256:   player $PLAYER_CHAR defined 
03EE:   player $PLAYER_CHAR controllable 
004D: jump_if_false -9 
0226: 0@ = actor $PLAYER_ACTOR health 
0AD1: show_formatted_text_highpriority "HEALTH: %D" time 3000 0@ 
0A93: terminate_this_custom_script 
015D: set_gamespeed 1.0 
0051: return
```

**Comment by XMDS** [comment]
@x87  It seems to be related to this update, there is no such problem in 3.6.0
https://github.com/sannybuilder/dev/issues/81

**Comment by x87** [comment]
@XMDS please test with this exe
https://sannybuilder.com/dev/sanny361.zip

**Comment by wmysterio** [comment]
> please test with this exe

This file not installable for me. Size is 1.1mb o_0

**Comment by wmysterio** [comment]
> 
> 
> please test with this exe

Work.

**Comment by x87** [comment]
Fixed in 3.6.2
